Popup documentation page is broken

### Steps
1. Load Popup docs page.

### Expected Result
It should load with all the documentation and examples.

### Actual Result
It loads the header and props but not the examples.
If it's a fresh load directly to http://react.semantic-ui.com/modules/popup then it's just a gray screen.

Here is the error in the console:
```
Uncaught Error: Cannot find module './modules/Popup/Types/PopupExample.js'.
    at app.e58e193a1f06c9b1affd.js:4169
    at o (app.e58e193a1f06c9b1affd.js:4169)
    at a (app.e58e193a1f06c9b1affd.js:4169)
    at ComponentExample._this.renderOriginalExample (app.e58e193a1f06c9b1affd.js:14)
    at ComponentExample.value (app.e58e193a1f06c9b1affd.js:14)
    at performInitialMount (react-dom.min.js:13)
    at p.mountComponent (react-dom.min.js:13)
    at Object.mountComponent (react-dom.min.js:14)
    at h.mountChildren (react-dom.min.js:14)
    at h._createInitialChildren (react-dom.min.js:13)

react-dom.min.js:13 Uncaught TypeError: Cannot read property '_currentElement' of null
    at p._updateRenderedComponent (react-dom.min.js:13)
    at p._performComponentUpdate (react-dom.min.js:13)
    at p.updateComponent (react-dom.min.js:13)
    at p.receiveComponent (react-dom.min.js:13)
    at Object.receiveComponent (react-dom.min.js:14)
    at Object.updateChildren (react-dom.min.js:13)
    at h._reconcilerUpdateChildren (react-dom.min.js:14)
    at h._updateChildren (react-dom.min.js:14)
    at h.updateChildren (react-dom.min.js:14)
    at h._updateDOMChildren (react-dom.min.js:13)
    at h.updateComponent (react-dom.min.js:13)
    at h.receiveComponent (react-dom.min.js:13)
```

I'm still able to switch to Popup.Content and Popup.Header but I can't go to another page. It'll change the address URL but nothing happens.

### Version
Current live version (0.62.3)
### Testcase
http://react.semantic-ui.com/modules/popup

--- a/docs/app/utils.js
+++ b/docs/app/utils.js
@@ -18,7 +18,7 @@ export const parentComponents = _.flow(
 /**
  * Get the Webpack Context for all doc site examples.
  */
-export const exampleContext = require.context('docs/app/Examples/', true, /(\w+Example\w+|index)\.js$/)
+export const exampleContext = require.context('docs/app/Examples/', true, /(\w+Example\w*|index)\.js$/)
 
 export const repoURL = 'https://github.com/Semantic-Org/Semantic-UI-React'
 export const semanticUIDocsURL = 'http://semantic-ui.com/'

